webapp/channels/src/components/post_edit_history/post_edit_history.tsx

```diff
@@ -12,6 +12,7 @@ import LoadingScreen from 'components/loading_screen';
 import EditedPostItem from './edited_post_item';
 
 import {PropsFromRedux} from '.';
+import alertIcon from 'images/icons/alert-icon.svg';
```

Comment:
use the existing one https://github.com/mattermost/mattermost-server/blob/master/webapp/channels/src/components/common/svg_images_components/alert_svg.tsx

Reply:
Done - however this version has a significant amount of whitespace padding so the width/height needed to be increased and padding was removed from the heading to account for this.